bindata/network/kuryr/001-rbac.yaml

```diff
@@ -19,13 +20,18 @@ rules:
   - update
   - patch
   - delete
```

Comment:
Is this needed?
https://github.com/openshift/cluster-network-operator/pull/1367/files#diff-b214c5fc4fe51960ff3611bc799ae5fd2815388ae066350a211c899dcec45068R18-R25

Reply:
Admission controller does DNS pod deletion to force them to run with TCP DNS: https://github.com/openshift/kuryr-kubernetes/blob/3385e1dbb0498e75ef2c03221f93228512505cbf/kuryr_kubernetes/cmd/webhook_server.py#L57-L59

I hate it, we don't support OSP13 in 4.10 anyway, but I'd rather not remove a "functionality" as a bugfix.